src/main/resources/assets/farmersdelight/lang/ru_ru.json

```diff
@@ -153,10 +153,10 @@
 
   "farmersdelight.tooltip.cooking_pot.empty": "Пусто",
   "farmersdelight.tooltip.cooking_pot.single_serving": "Содержит одну порцию",
-  "farmersdelight.tooltip.cooking_pot.many_servings": "Содержит %s порций",
+  "farmersdelight.tooltip.cooking_pot.many_servings": "Содержит порций: %s ",
```

Comment:
I think I might prefer the old sentence structure. Do you feel this one is gramatically better in Russian?